apps/browser/src/autofill/notification/bar.ts

```diff
@@ -30,7 +30,16 @@ function load() {
     notificationChangeDesc: chrome.i18n.getMessage("notificationChangeDesc"),
   };
 
-  document.getElementById("logo-link").title = i18n.appName;
+  const logoLink = document.getElementById("logo-link") as HTMLAnchorElement;
+  logoLink.title = i18n.appName;
+
+  // Update logo link to user's regional domain
+  const webVaultURL = getQueryVariable("webVaultURL");
+  const newVaultURL = webVaultURL && decodeURIComponent(webVaultURL);
+
+  if (newVaultURL && newVaultURL !== logoLink.href) {
```

Comment:
Nit - Please correct me if I'm wrong, but the logic here seems a bit more complicated than necessary?Couldn't you just do the following?

```suggestion
  const newVaultURL = decodeURIComponent(webVaultURL);

  if (newVaultURL !== logoLink.href) {
```

I might be wrong though, maybe there's some context I'm missing...

Either way, not blocking for this nitpick.

Reply:
I think I added this to guard against any overlooked cases where `webVaultURL` is missing; `getQueryVariable` _can_ return `null` in those cases, and `decodeURIComponent` will make `null` into a string value, which is truthy, resulting in the href updating to `'null'`.

We can use the simpler logic if we're confident we don't need that case guard, though.

Reply:
That makes sense, I think the guarded approach is worth keeping in place.